R/oauth.R

```diff
@@ -133,10 +133,30 @@ getTwitterToken <- function(tokenFileId="", useCache=TRUE){
   consumer_secret = "xYNUMALkRnvuT3vls48LW7k2XK1l9xjZTLnRv2JaFaM"
   appName = "twitter"
   endpointType = "twitter"
-  tokenFileName ="twitter_token.rds"
-  # Get OAuth credentials (For twitter use OAuth1.0)
-  twitter_token <- getOAuthToken(consumer_key, consumer_secret, appName, endpointType, "", tokenFileName, tokenFileId, useCache, version = "1.0")
-  twitter_token
+  # retrieve token info from environment
+  # main purpose is to enable server refresh
+  token_info <- getTokenInfo("twitter")
+  if(!is.null(token_info)){
+    HttrOAuthToken1.0$new(
+      request = "https://api.twitter.com/oauth/request_token",
+      authorize = "https://api.twitter.com/oauth/authenticate",
+      access = "https://api.twitter.com/oauth/access_token",
+      appname = "twitter",
+      key = consumer_key,
```

Comment:
please use obviously dummy key and secret here to avoid confusion, instead of using the ones for localhost solution.
Some comments that tells that this is different from the hardcoded ones above would help.